app/app.go

```diff
@@ -628,7 +631,34 @@ func New(
 		),
 	)
 
-	app.customKeepers(bApp, keys, appCodec, govRouter, homePath, appOpts, wasmOpts)
+	var err error
+	wasmDir := filepath.Join(homePath, "wasm")
+	app.wasmCfg, err = wasm.ReadWasmConfig(appOpts)
+	if err != nil {
+		panic(fmt.Sprintf("error while reading wasm config: %s", err))
+	}
+
+	// The last arguments can contain custom message handlers, and custom query handlers,
+	// if we want to allow any custom callbacks
+	availableCapabilities := "iterator,staking,stargate,cosmwasm_1_1"
```

Comment:
```suggestion
	availableCapabilities := "iterator,staking,stargate,cosmwasm_1_1,umee"
```
can you add `umee` also